docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.agent/tools-agent.md

```diff
@@ -64,6 +97,45 @@ Refine the Tools Agent node's behavior using these options:
 
 Refer to the main AI Agent node's [Templates and examples](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.agent/index/#templates-and-examples) section.
 
+## Dynamic parameters for tools
+
+When configuring tools connected to the Tools Agent, you can use the `$fromAI()` function to dynamically populate parameter values using the AI model. The AI model will fill in appropriate data given the context from the task and information from other connected tools.
+
+/// note | Only for the Tools Agent
+The `$fromAI()` function is only available for tools connected to the Tools Agent. You also can't use the `$fromAI()` function with the [Call n8n Workflow](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolworkflow/) tool.
+///
+
+The `$fromAI()` function accepts the following parameters:
+
+<!-- vale off -->
+
+| Parameter | Type | Required? | Description |
+| --------- | ---- | --------- | ----------- |
+| `key` | string | :white_check_mark: | A string representing the key or name of the argument. This must be between 1 and 64 characters in length and can only contain lowercase letters, uppercase letters, numbers, underscores, and hyphens. |
+| `description` | string | :x: | A string describing the argument. |
+| `type` | string | :x: | A string specifying the data type. Can be string, number, boolean, or json (defaults to string). |
+| `defaultValue` | any | :x: | The default value to use for the argument. |
+
+<!-- vale on -->
+
+As an example, you could use the following `$fromAI()` expression to dynamically populate a field with a name:
+
+```javascript
+$fromAI("name", "The commenter's name", "string", "")
```

Comment:
What's up with the empty string here as default? 🤔

Reply:
I guess that's not a very good default :-) Thanks for the heads up, I'll update.